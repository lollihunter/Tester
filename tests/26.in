20 4 20
1 1
3 4
14 1
16 1
2 1
11 2
20 2
14 4
2 4
3 1
9 3
8 1
19 2
13 2
4 2
14 2
2 3
17 1
15 3
20 4
