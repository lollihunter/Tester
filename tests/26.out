13
1 5 10 15 12 11 6 14 16 19 18 13 20 
